.github/workflows/performance.yml

```diff
@@ -70,16 +98,30 @@ jobs:
           docker-compose run --rm php php -i
           docker-compose run --rm php locale -a
 
-      - name: Install MU plugin
-        run: |
-          mkdir ./src/wp-content/mu-plugins
-          cp ./tests/performance/mu-plugins/server-timing.php ./src/wp-content/mu-plugins/server-timing.php
-
       - name: Install WordPress
         run: npm run env:install
 
+      - name: Install wordpress importer plugin and import dummy data
+        run: |
+          npm run env:cli -- plugin install wordpress-importer --activate
+          curl -O https://raw.githubusercontent.com/WPTT/theme-unit-test/master/themeunittestdata.wordpress.xml
+          npm run env:cli -- import themeunittestdata.wordpress.xml --authors=create
+          rm themeunittestdata.wordpress.xml
+
+      - name: Update permalink structure
+        run: |
+          npm run env:cli -- rewrite structure '/%year%/%monthnum%/%postname%/'
+
+      - name: Install MU plugin
+        run: |
+          mkdir ./build/wp-content/mu-plugins
+          cp ./tests/performance/mu-plugins/server-timing.php ./build/wp-content/mu-plugins/server-timing.php
+
       - name: Run Performance tests
         run: npm run test:performance
 
+      - name: Print Performance tests results
+        run: "node ./tests/performance/results.js"
```

Comment:
Just double-checking, this is temporary and will be replaced by sending the data to the Codehealth dashboard in a subsequent PR, correct?

Reply:
Thanks @felixarntz for asking. Yes we will sending this data to the Codehealth dashboard.